step-capstone/src/components/Flight.js

```diff
@@ -18,8 +18,8 @@ export default function Flight( data ) {
     return(
         <Grid item>
             <Typography variant="h6" gutterBottom>Flight from { data.departureAirport } to { data.arrivalAirport }</Typography>
-            <Typography variant="subtitle2" gutterBottom>Departure: { data.departureDate }</Typography>
-            <Typography variant="subtitle2" gutterBottom>Arrival: { data.arrivalDate }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Departure: { data.departureDate.toString() }</Typography>
+            <Typography variant="subtitle2" gutterBottom>Arrival: { data.arrivalDate.toString() }</Typography>
```

Comment:
I can't remember, but I thought we decided we would handle already be passing string versions of dates into our components?

Reply:
Yeah, so I realized that trip object actually needs to have the real date representation in order to be able to load the edit popover to display the previously selected date in the date input